vacancy.md

```diff
@@ -1,39 +1,60 @@
-# Правила публикации вакансий и резюме
+# Rules for postin job opportunities and CV | Правила размещения вакансий и резюме
 
-Эти правила помогают соискателям ориентироваться, а компаниям — привлекать людей. Публикация вакансий и резюме разрешается не чаще чем один раз в сутки.
+## Russian
 
+Эти правила помогают соискателям ориентироваться, а компаниям — привлекать людей. Публикация вакансий и резюме разрешается не чаще чем один раз в сутки.
 Посты нужно размещать в чате: если правила будут соблюдены, то админы опубликуют вакансию или резюме в канал.
 
-## Правила размещения вакансий
+### Правила размещения вакансий
 
+```bash
 #вакансия (хештег обязателен — без него бот не заработает)
-
 Город и адрес офиса: ... (район или метро)
-
 Формат работы: офис или удаленка
-
 Занятость: полная, частичная, проектная
-
 Зарплатная вилка: от ... до ...
-
 (сумма после налогов либо с ссылкой на калькулятор. «По договоренности» или «по результатам собеседования» не подходит, только конкретная вилка)
-
 Описание вакансии: ... (обязательно опишите текущий стек)
-
 Название компании: ...
-
 (с ссылкой; если у вас кадровое агентство — укажите название КА)
-
 Контакты: ...
+```
 
-## Правила размещения резюме
-
-#резюме либо #ищу (хештег обязателен — без него бот не заработает)
+### Правила размещения резюме
 
+```bash
+#резюме #ищу (хештег обязателен — без него бот не заработает)
 Формат работы: офис, удаленка
-
 Занятость: полная, частичная, проектная
-
 Ожидания по зарплате: от ...
-
-Обо мне: ссылка на резюме, Гитхаб или что посчитаете нужным
\ No newline at end of file
+Обо мне: ссылка на резюме, Гитхаб или что посчитаете нужным
+```
+
+## English
+
+These rules help applicants navigate and companies help attract people. The publication of vacancies and resumes is permitted no more than once a day. Posts need to be posted in the chat: if the rules are followed, then the admins will publish the vacancy or resume in the channel.
+
+### Правила размещения вакансий
+
+```bash
+#вакансия (a hashtag is required - without it, the bot will not work)
+City and office address: ... (district or metro)
+Work format: office or remote
+Employment: full, partial, project
+Salary plug: from ... to ...
+(amount after taxes or with reference to the calculator. “By agreement” or “by the results of the interview” is not suitable, only a specific plug )
```

Comment:
(amount after taxes or with reference to the calculator. “Depending on experience” or “based on the intervie result” is not suitable, only a specific range)